SuperHero/src/test/java/ApiTests.java

```diff
@@ -0,0 +1,280 @@
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.testng.annotations.BeforeClass;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static io.restassured.RestAssured.given;
+public class ApiTests {
+	private String baseUrl = "https://superhero.qa-test.csssr.com/superheroes";
+	@BeforeClass
+	public void setUp() {
+		RestAssured.baseURI = baseUrl;
+	}
+	@Test
+	@DisplayName("Add superhero with all required fields")
+	void postTest() {
+		String body = """
+				    {
+				        "birthDate": "2019-02-21",
+				        "city": "GOTHAM",
+				        "fullName": "BATMAN",
+				        "gender": "M",
+				        "mainSkill": "fight"
+				    }
+				""";
+		ValidatableResponse response =
+				given().
+						header("accept", "application/json").
+						header("Content-Type", "application/json").
+						body(body).
+						when().
+						post(baseUrl).then();
+		Response response1 = response.extract().response();
+		int status = response1.statusCode();
+		String responseBody = response1.body().prettyPrint();
+		Assertions.assertEquals(200, status, "Статус код не соответствует ожидаемому");
```

Comment:
It's better to follow AAA (Arrange - Act - Assert) pattern to organize the test.

First A (arrange) you are doing all setups and prerequisites.

Then after empty line you are doing Act - actual action of calling the POST method.

At Assert you are getting data and doing all your assertions.